src/data_snack/entities/compound.py

```diff
@@ -2,12 +2,12 @@
 from collections import ChainMap
 from dataclasses import dataclass
 from itertools import chain
-from typing import List, Any, Type
+from typing import List, Any
 
 from data_snack.entities import Entity
 from data_snack.entities.entity_meta import CompoundEntityMetaClass
 from data_snack.entities.models import SourceEntity
-from data_snack.entities.utils import map_values, filter_missing_values
+from data_snack.entities.utils import map_values, filter_missing_values, get_unique_values
 
 
```

Comment:
Those functions can be simplified

```
from itertools import chain
from typing import List


def get_unique_values(values: List[str]) -> List[str]:
    return list(set(values))


_map_unique_values = lambda mapping, fields: filter_missing_values(map_values(mapping, fields))


def get_all_fields(cls) -> List[str]:
    """Gets all CompoundEntity fields."""
    return get_unique_values(chain(*[
        _map_unique_values(source.source_fields_mapping, source.entity.get_all_fields())
        for source in cls.Meta.sources
    ]))
```

Reply:
name `_map_unique_values` doesn't fit since we use filter and map functions, wouldn't `_filter_mapped_values` be better?